centos_ci.sh

```diff
@@ -298,7 +298,7 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration_all
+  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="--tags=~coolstore&&~addon-xpaas"
```

Comment:
I recall an issue where we wanted to make sure that the target called by CI is a direct Makefile target and that we did not want to use custom properties within the CI script. This was to make it clear which features are run in CI jsut be looking at the Makefile. What happened to this issue?

Reply:
@hferentschik I just remember talk about integration_nightly/all, however the new target for CI seems reasonable for me. I will create it and will not use exclusion of tags, but instead will define the test directly tag by tag so it is easily understandable and does not face the issue with multiple ampersands. For now I will just mention in docs that defining multiple exclusive tags has issues and thus we recommend to define tags inclusively.

Possible follow up fix for making the tags more usable could be probably to move the tags from GODOG_OPTS to its own new parameter GODOG_TAGS.